.coveragerc

```diff
@@ -769,6 +769,7 @@ omit =
     homeassistant/components/microbees/button.py
     homeassistant/components/microbees/const.py
     homeassistant/components/microbees/coordinator.py
+    homeassistant/components/microbees/cover.py
```

Comment:
You should consider add tests

Reply:
actually they're not mandatory and we've speed up to introduce all our platforms, we will write in future releases